app/src/Homepage/Homepage.css

```diff
@@ -46,3 +46,78 @@
   height: 200vh;
   text-align: center;
 }
+
+/* Welcome header */
+.welcome {
+  text-align: center;
+  margin-top: 60px;
+  font-size: 300%;
+}
+
+/* How it works: header */
+.howitworks {
+  position: absolute;
```

Comment:
<img width="545" alt="Screen Shot 2023-11-02 at 12 39 09 AM" src="https://github.com/twang7310/rate-my-cse/assets/145406076/1f10b2c7-1c97-4797-8f38-ab640856ff0c">

Looks like this with smaller window

Recommendation 1:
Remove this position: absolute
That way, if Welcome expands, HowItWorks will move down because it's own 'block' that always takes up its own space (and won't overlap with anything)

Reply:
Fixed